src/Nri/Ui/SideNav/V3.elm

```diff
@@ -0,0 +1,530 @@
+module Nri.Ui.SideNav.V3 exposing
+    ( view, Config, NavAttribute
+    , navLabel
+    , navCss, navNotMobileCss, navMobileCss, navQuizEngineMobileCss
+    , entry, entryWithChildren, html, Entry, Attribute
+    , icon, custom, css, nriDescription, testId, id
+    , onClick
+    , href, linkSpa, linkExternal, linkWithMethod, linkWithTracking, linkExternalWithTracking
+    , primary, secondary
+    , premiumDisplay
+    )
+
+{-|
+
+
+# Changes from V1
```

Comment:
Do we keep these and add "Changes from V2?" or do we only do the most recent? Either way, we gotta add a thing here.